ansible_base/rbac/management/commands/RBAC_checks.py

```diff
@@ -0,0 +1,50 @@
+from collections import defaultdict
+
+from django.core.management.base import BaseCommand
+
+from ansible_base.rbac import permission_registry
+from ansible_base.rbac.models import ObjectRole, RoleDefinition
+
+
+class Command(BaseCommand):
+    help = "Runs bug checking sanity checks, gets scale metrics, and reccomendations for Role Based Access Control"
+
+    def handle(self, *args, **options):
+        rd_ct = RoleDefinition.objects.count()
+        self.stdout.write(f'Inspecting {rd_ct} role definitions')
+        self.stdout.write('  checking for minimum of view permission')
+        indexed_rds = defaultdict(list)
+        for rd in RoleDefinition.objects.prefetch_related('permissions'):
+            perm_list = list(rd.permissions.values_list('codename', flat=True))
+            if 'view' not in ' '.join(perm_list):
+                self.stdout.write(f'Role definition {rd.name} does not list any view permissions and this is considered invalid')
+            perm_set = frozenset(perm_list)
+            indexed_rds[perm_set].append(rd)
+
+        self.stdout.write('  checking for duplicate role definitions')
+        for perm_set, rd_list in indexed_rds.items():
+            if len(rd_list) > 1:
+                self.stdout.write('Found duplicate role definitions with same permissions list:')
```

Comment:
Minor: we should probably make use of `self.style.*` for many of these.  This one seems like `self.style.NOTICE` or `self.style.WARNING`.

https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/#django.core.management.BaseCommand.style